src/config.py

```diff
@@ -143,6 +148,10 @@ class Status:
 
         # TODO Future PR add more status messages here as constants
         UNHEALTHY_UPGRADE = BlockedStatus("Unhealthy after upgrade.")
+        INCOMPATIBLE_UPGRADE = BlockedStatus(
+            "Refresh incompatible. Rollback to previous revision with `juju refresh`"
+        )
+        WAITING_POST_UPGRADE_STATUS = MaintenanceStatus("Waiting for post upgrade checks")
```

Comment:
IIRC this must be of type MaintenceStatus (for the reasons we discussed in our 1:1) ?

if not-WaitingStatus would be more appropriate

Reply:
It can be whatever I want it to be :)
If you think WaitingStatus is better, we let's do it